README.md

```diff
@@ -12,37 +12,46 @@ This role requires the [php-fpm role](https://github.com/stuvusIT/php-fpm), the
 
 ### General
 
-| Name                 | Required                 | Default        | Description                                            |
-|----------------------|:------------------------:|----------------|----------------------------------------------------------------|
-| `wordpress_mysql_password`      | :heavy_check_mark:       |         | The password for the mysql database                             |
-| `wordpress_default_user` | :heavy_multiplication_x:       | `admin`        | This is the user that is created on setup |
-| `wordpress_default_password`      | :heavy_check_mark:|                | The password that is used for the default user |
-| `wordpress_default_email`      | :heavy_check_mark:|                | The email address for the default user |
-
+| Name                         | Required/Default   | Description                                                                   |
+|------------------------------|:------------------:|-------------------------------------------------------------------------------|
+| `wordpress_mysql_password`   | :heavy_check_mark: | The password for the mysql database                                           |
+| `wordpress_default_user`     | `admin`            | This is the user that is created on setup                                     |
+| `wordpress_default_password` | :heavy_check_mark: | The password that is used for the default user                                |
+| `wordpress_default_email`    | :heavy_check_mark: | The email address for the default user                                        |
+| `wordpress_plugin_merge`     | `true`             | The email address for the default user                                        |
+| `wordpress_plugins`          | `[]`               | List of plugins to install and options to set. For more information see below |
+
+### Plugins
+
+| Name      | Required/Default   | Description                                                             |
+|-----------|--------------------|-------------------------------------------------------------------------|
+| `name`    | :heavy_check_mark: | The name(slug) of the plugin to be installed.                           |
+| `merge`   | `true`             | Setting if the options should be merged or overwritten                  |
```

Comment:
I think the default is rather `{{ wordpress_plugin_merge }}`.

Reply:
fixed